tools/ansible/tasks/wsu/main.yaml

```diff
@@ -73,16 +75,50 @@
         dest: "{{ win_temp_dir.path }}\\worker.ign"
         validate_certs: no
 
+    - name: Get kubernetes version
+      delegate_to: localhost
+      shell: "oc version -o json | jq -r '.serverVersion.minor' | tr -d +"
+      register: kubernetes_version
+
+    # This translates a kubernetes version to an OpenShift version. This works under the assumption that each OpenShift
+    # release corresponds with a kubernetes release. This is being done this way, and not with oc get clusterversion,
+    # as OpenShift CI doesn't have the actual version attached to its clusters, instead replacing it with 0.0.1 and
+    # information about the release creation date
+    - name: Get cluster version
+      delegate_to: localhost
+      shell: "MINORVERSION={{ kubernetes_version.stdout }} BASEK8SVERSION=16 BASEOPENSHIFTVERSION=3; if [ \"$MINORVERSION\" -lt \"$BASEK8SVERSION\" ]; then echo unsupported kubernetes version; exit 1; fi; VERSIONINCREMENTS=$((MINORVERSION - BASEK8SVERSION)); echo 4.$((BASEOPENSHIFTVERSION + VERSIONINCREMENTS)); exit 0"
+      register: cluster_version
+
+    # curl -s https://api.github.com/repos/openshift/windows-machine-config-operator/releases gets the list of all
+    # releases, we use the cluster version from previous ansible task to filter out the release we need.
+    # The wmcb release tags are supposed to confirm to the following semver format.
+    # v<openshift-major-version>.<openshift-major-version>.<wmcb_build_number>-<release_type>. For example, v4.4-1-alpha.
+    # For a <openshift-major-version-version>.<openshift-minor-version>, there can be many wmcb builds available, for
+    # example, OpenShift 4.4 can map to v4.4.1-alpha, v4.4.2-alpha, we will always pick up the latest build available
+    # which is v4.4.2-alpha,
+    # In future, when we cut beta and GA releases, the playbook should always go for GA version when available.
+    - name: Get hybrid overlay url
+      delegate_to: localhost
+      shell: "curl -s https://api.github.com/repos/openshift/windows-machine-config-operator/releases | jq '.[] | select (.tag_name == '$(curl -s https://api.github.com/repos/openshift/windows-machine-config-operator/releases |  jq '.[]' | jq '.tag_name' | grep -i {{ cluster_version.stdout }} | sort -r | awk 'NR==1')' )' | jq ' .assets[] | select (.name == \"{{ hybrid_overlay_exe }}\" )'| jq -r .browser_download_url"
```

Comment:
Is it possible to curl this only once to an environment variable and then run `jq` against the variable?